lib/travis/build/templates/header.sh

```diff
@@ -52,9 +53,12 @@ travis_cmd() {
   fi
 
   if [[ -n "$retry" ]]; then
-    travis_retry eval "$cmd"
+    travis_retry eval "$cmd $secure"
   else
-    eval "$cmd"
+    eval "$cmd $secure"
+    if [[ -n $secure && $? -ne 0 ]]; then
+      echo "The previous command failed"
```

Comment:
The white spaces are special in bash, since they delimit commands and arguments. The leading white spaces here are particularly harmful, since we end up executing something like:

```
export SECRET=    12345678-super-duper-secret
```

The leading numeral makes this an invalid token, which would be dumped to STDERR. We want to avoid that.

Perhaps we can show examples of "special characters": `…special characters such as ' ' and '$'.`.